portability-types-common/src/main/java/org/datatransferproject/types/common/models/DataVertical.java

```diff
@@ -0,0 +1,46 @@
+package org.datatransferproject.types.common.models;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Represents the data type of a transfer job
+ */
+public enum DataVertical {
```

Comment:
```suggestion
// TODO consider if we can reference data models directly rather than enums; for now beware that each item in this enum is represented by one and only one common data model.
public enum DataVertical {
```